cmd/deploy.go

```diff
@@ -358,7 +358,21 @@ func autofillApimodel(dc *deployCmd) error {
 
 	if k8sConfig != nil && k8sConfig.Addons != nil && k8sConfig.IsContainerMonitoringAddonEnabled() {
 		log.Infoln("container monitoring addon enabled")
-		err := dc.configureContainerMonitoringAddon(ctx, k8sConfig)
+		var workspaceDomain string
+		cloudspecConfig := dc.containerService.GetCloudSpecConfig()
+		log.Infoln("selected cloud name:", cloudspecConfig.CloudName)
```

Comment:
Let's get rid of this log statement, as we don't need to output this for all cluster deploy operations

Reply:
> Let's get rid of this log statement, as we don't need to output this for all cluster deploy operations

Fixed. removed the log statement.